File: src/main/java/com/nedap/university/go/gocommands/servercommands/TableFlippedCommand.java
package com.nedap.university.go.gocommands.servercommands;

import com.nedap.university.go.client.ServerHandler;
import com.nedap.university.go.gocommands.Command;

/**
 * Created by martijn.slot on 21/02/2017.
 */
public class TableFlippedCommand extends Command {

    public TableFlippedCommand(String[] splitMessage, ServerHandler serverHandler) {
        super();
    }

    @Override
    public void execute() {

    }

    @Override
    protected void cannotExecute() {
        return null;
    }
}

File: src/main/java/com/nedap/university/go/gocommands/serverToClient/TableFlippedCommand.kt
package com.nedap.university.go.gocommands.serverToClient

import com.nedap.university.go.client.ServerHandler
import com.nedap.university.go.gocommands.Command

/**
 * Created by martijn.slot on 21/02/2017.
 */
class TableFlippedCommand(splitMessage: Array<String>, private val serverHandler: ServerHandler) : Command() {

    init {
        this.splitMessage = splitMessage
    }

    @Override
    fun execute() {
        serverHandler.handleTableFlipped(splitMessage)

    }
}
